lighthouse-core/gather/connections/connection.js

```diff
@@ -100,7 +100,8 @@ class Connection {
       // handleRawError returns or throws synchronously; wrap to put into promise chain.
       return callback.resolve(Promise.resolve().then(_ => {
         if (object.error) {
-          return this.handleRawError(object.error, callback.method);
+          log.formatProtocol('method <= browser ERR', {method: callback.method}, 'error');
+          throw new Error(`Protocol error (${callback.method}): ${object.error.message}`);
```

Comment:
definitely inconsistent. From #861 is this cs link, currently showing only `DOM` complains if you disable when already disabled: https://cs.chromium.org/search/?q=if.*!enabled+file:%5Esrc/third_party/WebKit/Source/core/inspector/+case:yes&sq=package:chromium&type=cs

@patrickhulce @paulirish if they want to pursue a debugger protocol bug :)